projects/ddata-ui-input/src/public-api.ts

```diff
@@ -22,7 +22,9 @@ export * from './lib/components/color/color-input.component';
 export * from './lib/components/date/date-input.component';
 export * from './lib/components/input/input.component';
 export * from './lib/components/search/search.component';
+export * from './lib/components/select/autocomplete-select/autocomplete-select.component';
```

Comment:
This component isn't public, it's available only over the select component with `autocomplete` mode.